sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -331,7 +331,8 @@ def _adjust_pipeline_for_dataflow_v2(self, pipeline):
             not pipeline._options.view_as(
                 TypeOptions).allow_non_deterministic_key_coders))
 
-  def run_pipeline(self, pipeline, options, pipeline_proto=None):
+  def run_pipeline(
+      self, pipeline, options, pipeline_proto=None, display_data=None):
```

Comment:
Should display_data be present on pipeline and/or pipeline_proto, rather than requiring we pass it separately?